CHANGELOG.md

```diff
@@ -18,6 +24,7 @@ All notable changes to this project will be documented in this file.
 
 - Processing of corrupted log events fixed; If errors occur, the error
   messages are added to the log event ([#583]).
+- Bundle builder should no longer keep serving deleted rules until it is restarted ([#578]).
```

Comment:
Can you please move this to `Unreleased` above?

Reply:
Goddamnit it keeps happening.. 412e7132b00577917df86f8e998a6856fa44d74f

Reply:
Happens to the best :)